eng/scripts/Language-Settings.ps1

```diff
@@ -33,9 +33,24 @@ function Get-python-AdditionalValidationPackagesFromPackageSet {
     return $startsWith
   }
 
+  $changedServices = @()
+  foreach($file in $diffObj.ChangedFiles) {
+    $pathComponents = $file -split "/"
+    # handle changes only in sdk/<service>/<file>/<extension>
+    if ($pathComponents.Length -eq 3 -and $pathComponents[0] -eq "sdk") {
+      $changedServices += $pathComponents[1]
+    }
+
+    # handle any changes under sdk/<file>.<extension>
+    if ($pathComponents.Length -eq 2 -and $pathComponents[0] -eq "sdk") {
```

Comment:
Isn't this already handled somewhere as a fallback to ensure we always have at least this package?

Reply:
A fallback for `sdk/*` paths or a generic fallback? Are there any cases where we don't want to run the pullrequest pipeline at all, as a way of blocking check enforcer for changes that need a pipeline implemented but don't have one (e.g. maybe outside of sdk).

Reply:
I thought we had a fallback that if there are no packages detected at all then we build template. I don't think we are at a place yet where we want to avoid running anything.

Reply:
> Isn't this already handled somewhere as a fallback to ensure we always have at least this package?

It is not currently. This PR handles any changes in `sdk/file.extension` or `sdk/blah/file.extension`. 

Existing coverage before this pr was relying on coverage from A) in direct package  changes or B) the handling of the other directories elsewhere in the Language-Settings file.